in_toto/envelope_test.go

```diff
@@ -192,12 +202,31 @@ func TestEnvelopeSign(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if err := env.Sign(key); err != nil {
-		t.Error(err)
-	}
+	t.Run("valid ed25519 key", func(t *testing.T) {
+		if err := env.Sign(key); err != nil {
+			t.Error(err)
```

Comment:
nit: add a return if there is an error as you don't want the test to continue. or a `t.Fatal`